easybuild/easyconfigs/c/CUDAcore/CUDAcore-11.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "EB_CUDA"
+name = 'CUDAcore'
+version = '11.0.2'
+local_nv_version = '450.51.05'
```

Comment:
update to `11.1.0_455.23.05`?

Reply:
@bartoldeman bumped to CUDA 11.1.0 in https://github.com/ComputeCanada/easybuild-easyconfigs/pull/13

Reply:
No ?

Reply:
As commented on the PR on your PR, we had already settled on 11.0.2 for 2020a... why change now ? This won't work as we have already software installed with 2020a = 11.0.2 for CUDA.

Reply:
Also, bumping it to 11.1.0 basically means it can't be used on any of our clusters yet, since the driver requirements for 11.0 just got met by our clusters.

Reply:
@mboisson I understand that this is a concern for ComputeCanada, but in the central repo we haven't settled on a CUDA version for `2020a` yet...

Are there other reasons to stick to an older CUDA version?

@akesandgren, @Micket, @lexming Thoughts on this?

Reply:
2020a should use tools that were released around 2020a (i.e. first half of 2020) not that were released in the late 2020... if this PR had extended until 2022, would we release 2020a with CUDA 13.0 ? I hope not...

Reply:
I discussed this a bit with @mboisson, and I can see why motivating to go with CUDA 11.1 now for the `2020a` generation is difficult to motivate, given the significant delay in settling on `fosscuda/2020a`.

Let's keep CUDA 11.1 for `fosscuda/2020b`, since it is compatible with GCC 10 according to the [release notes](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html)